hassio-google-drive-backup/DOCS.md

```diff
@@ -0,0 +1,172 @@
+
+### Option: `snapshot_password`
+
+When set, snapshots are created with a password. You can use a value from your secrets.yaml by prefixing the password with "!secret". You'll need to remember this password when restoring snapshots.
+
+> Example: Use a password for snapshot archives
+>
+> ```json
+> "snapshot_password": "super_secret"
+> ```
+>
+> Example: Use a password from secrets.yaml
+>
+> ```json
+> "snapshot_password": "!secret snapshot_password"
+> ```
+
+### Option: `snapshot_name` (default: "{type} Snapshot {year}-{month}-{day} {hr24}:{min}:{sec}")
+
+Sets the name for new snapshots. Variable parameters of the form `{variable_name}` can be used to modify the name to your liking. A list of available variables are given [here](https://github.com/sabeechen/hassio-google-drive-backup#can-i-give-snapshots-a-different-name).
+
+### Option: `generational_*`
+
+When set, older snapshots will be kept longer using a [generational backup scheme](https://en.wikipedia.org/wiki/Backup_rotation_scheme). See the [question here](https://github.com/sabeechen/hassio-google-drive-backup#can-i-keep-older-backups-for-longer) for configuration options.
+
+### Option: `exclude_folders`
+
+When set, excludes the comma separated list of folders by creating a partial snapshot.
+
+### Option: `exclude_addons`
+
+When set, excludes the comma separated list of addons by creating a partial snapshot.
+
+_Note_: Folders and add-ons must be identified by their "slug" name. It is recommended to use the `Settings` dialog within the add-on web UI to configure partial snapshots since these names are esoteric and hard to find.
+
+### Option: `enable_snapshot_stale_sensor` (default: True)
+
+When false, the add-on will not publish the [binary_sensor.snapshots](https://github.com/sabeechen/hassio-google-drive-backup#how-will-i-know-this-will-be-there-when-i-need-it) stale sensor.
+
+### Option: `enable_snapshot_state_sensor` (default: True)
+
+When false, the add-on will not publish the [sensor.snapshot_state](https://github.com/sabeechen/hassio-google-drive-backup#how-will-i-know-this-will-be-there-when-i-need-it) sensor.
+
+### Option: `notify_for_stale_snapshots` (default: True)
+
+When false, the add-on will send a [persistent notification](https://github.com/sabeechen/hassio-google-drive-backup#how-will-i-know-this-will-be-there-when-i-need-it) in Home Assistant when snapshots are stale.
+
+---
+
+### UI Server Options
+
+The UI is available through Home Assistant [ingress](https://www.home-assistant.io/blog/2019/04/15/hassio-ingress/).
+
+It can also be exposed through a webserver on port `1627`, which you can map to an externally visible port from the add-on `Network` panel. You can configure a few more options to add SSL or require your Home Assistant username/password.
```

Comment:
From the old `README.md`:
> The UI is exposed through a webserver on port 1627, which you can map to an externally visible port from the add-on "Network" panel, which also defaults to port 1627.  You can configure a few more options to add SSL or require your Home assistant username/password.  In a future release, the add-on will support [ingress](https://www.home-assistant.io/blog/2019/04/15/hassio-ingress/) but for now it is disabled for compatibility reasons (see [this issue](https://github.com/sabeechen/hassio-google-drive-backup/issues/19)).

This issue has been closed, so that text has also be changed accordingly!